src/lib/loaders/loaders_with_authentication/gravity.ts

```diff
@@ -220,6 +220,7 @@ export default (accessToken, userID, opts) => {
       { method: "DELETE" }
     ),
     notificationsFeedLoader: gravityLoader("me/notifications/feed"),
+    partnerAllLoader: gravityLoader((id) => `partner/${id}/all`),
```

Comment:
Need to add a new authenticated loader to the `partner/:id/all` endpoint to get vat data.

Reply:
(not related to this PR) haha I was looking if we have this `all` naming pattern anywhere else and found [this loader](https://github.com/artsy/metaphysics/blob/master/src/lib/loaders/loaders_with_authentication/gravity.ts#L239) this loaded that has `All` in the name but doesn't really call the `all` endpoint 😄

Reply:
Yeah potentially it was trying to differentiate from this [sale artworks loader](https://github.com/artsy/metaphysics/blob/e037268902b9a2da332dc211ae178f503e6a724c/src/lib/loaders/loaders_with_authentication/gravity.ts#L245).

We probably don't have many examples because Volt wasn't using MP at the time so there was no need, and now a lot of stuff are done through stitching.